ls1intum/artemis-helm
# `Feature`: Horizontal Pod Autoscaling

Support horizontal auto scaling based on user count
- Installs a prometheus server and an adapter to use the collected metrics within Kubernetes
- Configures a horizontal autoscaler that scales the Deployment
- Configures Artemis to expose the metrics


Testing steps:
1. Install dependencies using `helm dep up` within the `artemis` directory
2. Enable autoscaling in the values.yml
3. Upgrade the helm deployment
4. After some time (~2 minutes due to the prometheus server/adapter starting), connect with multiple users to the ingress
5. The deployment should scale according to the configured user count
